### ISSUE TITLE
CircleCI test summary badge

### ISSUE THREAD

**Issue by paulmelnikow** [badges/shields#2642]
It would be great to show something like **1457 passing, 1 failing** for our daily tests badge.

See https://github.com/badges/shields/pull/2526#issuecomment-447207239

**Comment by paulmelnikow** [comment]
Tomorrow I'm going to pair with @jankeromnes to do some triage on #2783, and we're going to try to tackle this as a new badge, and probably refactor one old one, too. All using Gitpod!

**Comment by calebcartwright** [comment]
Does Gitpod have any live/code share features? I think I remember seeing elsewhere an idea for doing a distributed hackathon leveraging Gitpod too. Might be cool if folks could share their session with each other and/or maintainers to help answer any questions, etc.

**Comment by paulmelnikow** [comment]
Yea, @jankeromnes said there is a way to share your screen. We could also screen-record these. Feel like we're trying, at this point, to gather feedback about the experience, though once we're feeling like it's in a good place we should definitely live-stream and post a recording of one of these!

**Comment by jankeromnes** [comment]
> Does Gitpod have any live/code share features?

Yes, I think you can share your workspace for live collaboration/coding, but I’ve never tried it yet. Tomorrow will be a first. 🙂

And it’s indeed a life-saving feature when you just can’t get something to work, and sending comments back and forth with a maintainer doesn’t cut it. Giving them access to your environment is really powerful.
